CONTRIBUTING.md

```diff
@@ -34,6 +34,20 @@ The acceptance tests require a Kubernetes cluster with a configured `kubectl`.
   brew install kubernetes-helm
   ```
 
+### Helm 2/3
+These tests will work with both Helm 2 and 3 if run through `bats`, e.g. `bats ./test/unit`. If copying the
+test command and running yourself, e.g.
+```sh
+helm template \
+  -s templates/sync-catalog-deployment.yaml  \
+  --set 'syncCatalog.enabled=true' \
+  --set 'syncCatalog.toConsul=false' \
+  .
+```
+It's expected that `helm` is Helm 3.
```

Comment:
Do you mean that the installed `helm` binary on your PATH should be helm 3?

Reply:
Yes, will make more clear.